pcsx2/x86/newVif_Unpack.cpp

```diff
@@ -73,8 +73,21 @@ static const __aligned16 Fnptr_VifUnpackLoop UnpackLoopTable[2][2][2] = {
 
 nVifStruct::nVifStruct()
 {
-	vifBlocks	=  NULL;
-	numBlocks	=  0;
+	vifBlocks = NULL;
+	block.startPtr = NULL;
+	recReserve = NULL;
+	recWritePtr = NULL;
+	idx = 0;
+	bSize = 0;
+	bPtr = 0;
+	numBlocks = 0;
+	block.num = 0;
```

Comment:
I think there are better alternative to init blocks struct
* could be memset
* could be initializer list: `= {}`
* Others?